src/test/java/org/fundacionjala/coding/maria/Kata8Test.java

```diff
@@ -0,0 +1,69 @@
+package org.fundacionjala.coding.maria;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Kata 8.
+ */
+public class Kata8Test {
+    private Kata8 kata8;
+
+    /**
+     * initialize kata8.
+     */
+    @Before
+    public void initialize() {
+        kata8 = new Kata8();
+    }
+
+    /**
+     * testBankOCRNormal test 1.
+     */
+    @Test
+    public void testBankOCRNormal() {
+        String filaUno =  "    _  _     _  _  _  _  _ ";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "    _  _     _  _  _  _  _ " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)